### PR TITLE
Fonts setup

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Adds the DS fonts.

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -2,15 +2,41 @@ import type { AppProps } from "next/app"
 // ChakraProvider import updated as recommended on https://github.com/chakra-ui/chakra-ui/issues/4975#issuecomment-1174234230
 // to reduce bundle size. Should be reverted to "@chakra-ui/react" in case on theme issues
 import { ChakraProvider } from "@chakra-ui/provider"
+import { Inter } from "next/font/google"
+import { IBM_Plex_Mono } from "next/font/google"
 
 // Chakra custom theme
 import theme from "@/@chakra-ui/theme"
 
+const inter = Inter({
```
Comment (nhsz) [inline src/pages/_app.tsx]: we could move `inter` and `mono` declarations into `@lib/fonts` to clean `_app` a bit

**Review by nhsz — CHANGES_REQUESTED** [review]
asking for a small change, looks good!

File: src/pages/_app.tsx (hunk 1 above)
Comment (pettinarip) [reply src/pages/_app.tsx]: Agreed.

**Comment by pettinarip** [comment]
@nhsz check the last commit, I removed the `display` value since the default is already `swap`. And removed the `weight` attr from the Inter font to make use of the variable font version of it (which is faster since we need to download only one file/font). The IBM font doesn't offer a variable font type so, in that case, we are forced to specify the weight.

**Review by nhsz — APPROVED** [review]
LGTM!

**Pull request merged**